docs/kafka/rhoas-cli-getting-started-kafka/README.adoc

```diff
@@ -180,7 +180,7 @@ You use these credentials and the bootstrap server URL to connect your applicati
 . Set the permissions for your service account to access the Kafka instance resources.
 +
 --
-This example allows applications to produce and consume messages from topics starting with a specified prefix that belong to any consumer group.
+This example allows applications to produce and consume messages from topics starting with a specified prefix that belong to any consumer group. The service account information you input here is the client ID credentials from the previous step.
```

Comment:
credentials would be typically associated to username and password. Skipping that word will be better.
```suggestion
This example allows applications to produce and consume messages from topics starting with a specified prefix that belong to any consumer group. The service account information you input here is the clientID from the previous step.
```

Reply:
Thanks Wojciech!